src/main/resources/avro/bdg.avdl

```diff
@@ -57,8 +57,55 @@ record Contig {
   union { null, string } species = null;
 }
 
+record Sequence {
+  union { null, string } bases;
+  union { null, string } qualities;
+}
+
+/**
+   The DNA fragment that is was targeted by the sequencer, resulting in
+   one or more reads.
+*/
+record Fragment {
+  /**
+     The name of this Fragment.
+  */
+  union { null, string } readName = null;
+
+  union { null, string } instrument;
+	union { null, string } runId;
+  /**
+     The following are Illumina specific, is it a problem?
```

Comment:
Suggestions?  We could consider throwing things such as `tile`, `xPos` and `yPos` into a map.

The one field I think needs to be kept is `lane`, as it's needed to segment the data in the flowcell.  Though not all sequencing tech has flowcells that are split into "lanes" (only Illumina and SOLiD [apparently](https://genohub.com/ngs-instrument-guide/)), the concept also exists for 454 (they can use different regions of the PTP to sequence different samples).  A good decision might be to change the `lane` field into a more generic `region` and change the type into a `string`, so we can represent any id.